addons/no-sprite-confirm/addon.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Remove sprite deletion confirmation",
+  "description": "Removes the confirmation prompt that asks if you are sure you want to delete a project sprite.",
+  "info": [
+    {
+      "text": "Tip: If you accidentally delete a sprite, you can undo the deletion by clicking Edit in the menu bar then Restore.",
```

Comment:
I think it's better if we reuse the `hide-delete-button` tip copied word-by-word